Packages/ca.tekly.logger/Runtime/TkLogMessage.cs

```diff
@@ -112,7 +112,11 @@ public void Print(StringBuilder sb)
         private void CopyCommonFields(int startIndex)
         {
             TkLogger.UpdateCommonProperties();
-            
+
+            if (this.Params.Length < startIndex + TkLogger.CommonFields.Count) {
```

Comment:
This shouldn't be necessary, where `Params` is created accounts for the count of CommonFields.

If the size isn't correct before calling this then that constructor should be fixed.

Also the are two `this.` in here. How dare you.